en/docs/ipaas/micro-integrator/developing-integrations-with-integration-studio.md

```diff
@@ -0,0 +1,106 @@
+# Developing Integrations with Integration Studio
+
+Choreo iPaaS offers a platform for developing and deploying integrations with ease. With the Integration Studio,
+developers can create integrations using WSO2 Synapse and deploy them in a Micro Intgrator (MI) runtime. Choreo iPaaS
+provides a centralized management system for integrations, making deployment and maintenance simple.
+
+## Integration Types
+
+Choreo supports following types of MI Integrations
+
+- __Integration as an API__
+- __Event-Triggered Integration__
+- __Scheduled Integration__
+- __Manually-Triggered Integration__
+- __Composite Integration (Coming soon)__
+
+## Develop with Integration Studio
+
+[Integration Studio](https://wso2.com/integration/integration-studio/) is a graphical development environment for
+building and deploying integration artifacts in WSO2 Micro Integrator. It is an Eclipse-based IDE that provides a
+comprehensive set of tools for designing and testing integration flows and building integration artifacts. With WSO2
+Integration Studio, developers can create integration projects and develop integration artifacts in a visual,
+drag-and-drop manner, reducing the time and effort required to build and deploy integration flows. Additionally, the
+Studio provides features such as debugging, testing, and version control to help ensure the quality and reliability of
+integration flows.
+
+Creating an Integration project and Composite Exporter project in WSO2 Integration Studio allows developers to export
+their integration projects as a single deployment artifact, called a composite application. This composite application
+contains all the components and configurations required to deploy the integration project to WSO2 Micro
+Integrator. The Composite Exporter project helps to simplify the deployment process by allowing developers to package
+their integration projects as a single, self-contained artifact that can be easily deployed and managed. This can help
+improve the consistency and reliability of integration projects and simplify the overall deployment process.
+
+To get started with Integration Studio, see [WSO2 API Manager Documentation - Developing Integration Solutions](https://apim.docs.wso2.com/en/latest/integrate/develop/intro-integration-development/).
+
+## Integration Project Directory Structure
+
+Integration Project is a multi module maven project. It can contain multiple modules with WSO2 Synpase configurations
```

Comment:
Suggestion:
"Integration Project is a multi-module maven project. It can contain multiple modules with WSO2 Synpase configurations and a composite application module to export all the configurations as a single deployable artifact.

In addition to the above, yopu can optionally add the OpenAPI definition file and libraries. You can add the OpenAPI definition to any location in the project, and you need to define its path when creating a component. You can add Java libraries to the `libs` directory in the project root."